docs/submitting.md

```diff
@@ -222,7 +222,7 @@ There are no fees for submitting or publishing in JOSS. You can read more about
 
 ## Preprint Policy
 
-Authors are welcome to submit their papers to a preprint server such as [arXiv](https://arxiv.org/), [bioRxiv](https://www.biorxiv.org/), [SocArXiv](https://socopen.org/), [PsyArXiv](https://psyarxiv.com/) at any point during the submission and review process.
+Authors are welcome to submit their papers to a preprint server (e.g., [arXiv](https://arxiv.org/), [bioRxiv](https://www.biorxiv.org/), [SocArXiv](https://socopen.org/), [PsyArXiv](https://psyarxiv.com/)) at any point during the submission and review process.
```

Comment:
I would like to somehow indicate that these are just some options, which is what I thought the eg and parens would do, but I'm not sure that succeeded.  I know it's not correct grammatically, but perhaps add ", etc." at the end of the list?